src/audio/Audio.js

```diff
@@ -171,7 +171,7 @@ class Audio extends Object3D {
 
 		if ( this.source !== null ) {
 
-			this.source.stop();
+			this.source.stop( this.context.currentTime + this.offset + delay );
```

Comment:
I wonder if it is correct to honor `offset` here. It is actually only intended for the start.

What was your motivation to add it in `stop()`?

Reply:
I was confused because offset is a part of Audio object
In WebAudio it is a start method parameter, not a field of AudioNode
https://developer.mozilla.org/en-US/docs/Web/API/AudioBufferSourceNode/start

So if we want to follow WebAudio it makes sense to ignore offset

Reply:
offset was removed